lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 423 bestuursorgaan detail page

## Description

Updated the designs for the organen/orgaan routes. Mostly design updates, but also remove the bestuursperiod selector here, made the bewerk a modal and make it possible to edit mandaten in the table.

<img width="1710" alt="Screenshot 2024-05-10 at 13 17 07" src="https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/ffadebef-c63a-4388-b427-a8ad8f3f8bec">
<img width="1709" alt="Screenshot 2024-05-10 at 13 17 26" src="https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/9508f9e4-b062-4c08-ac9e-dc92e9eed858">
<img width="828" alt="Screenshot 2024-05-10 at 13 17 35" src="https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/d29ef73a-2231-415e-b062-b9c3ebe76f7e">

## How to test

Check the index page of a specific bestuursorgaan, best to check both an active, inactive and a decretaal one.
